src/Components/Dialogs/Dialogs.tsx

```diff
@@ -1,7 +1,6 @@
 import React from 'react';
 import DialogMenu from './Menu/Menu';
 import DialogContent from './Content/Content';
-import DialogMedia from './Media/Media';
 import DialogEdit from './Edit/Edit';
```

Comment:
`import DialogEdit from './Edit';` would also work as it will automatically grab the _index.js_. Not necessary, but nice to know. ;-)